### PR TITLE
Add visit_IfExp and tests

### PR DESCRIPTION
**Pull request by AgathiyanB**
Add 'visit_IfExp' method

## Description
Added 'visit_IfExp' method and changed the '_handle_conditional_node' method adding cases for the new method. Tests for the new method have also been added.

## Related Issue
#178 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation in the README file.
- [ ] I have updated the documentation accordingly.
- [x] I have added an entry in CHANGELOG.md.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jendrikseipp** [comment]
Thanks for the PR! 

@RJ722 could you review the code, please?

File: tests/test_conditions.py [hunk 1]
```diff
@@ -212,3 +212,20 @@ def test_redundant_if(v):
...
+    )
+    check_unreachable(v, 1, 1, "ternary")
+
+def test_if_exp_false(v):
+    v.scan(
+        """\
+foo if False else bar
+"""
+    )
+    check_unreachable(v, 1, 1, "ternary")
```
[the remark this answers is not in the document]
Comment (AgathiyanB) [reply tests/test_conditions.py]: I've amended the commit and pushed it with the newline a the end of the file now. Will that have amended this PR as well or should this be fine?

[the remark this answers is not in the document]
Comment (RJ722) [reply tests/test_conditions.py]: Working, wonderful!

**Comment by AgathiyanB** [comment]
Sorry, there was a merge conflict in CHANGLELOG.md, I think it should be resolved now.

**Comment by jendrikseipp** [comment]
Thanks!

**Pull request merged**